samples/nrf9160/modem_shell/README.rst

```diff
@@ -720,6 +784,18 @@ The LEDs have the following functions:
 LED 3 (nRF9160 DK)/Blue LED (Thingy:91):
    Indicates the LTE registration status.
 
+Power measurements
+==================
+
+Power measurements can be conducted using `Power Profiler Kit II (PPK2)`_.
+To achieve proper power measurement numbers, UART must be disabled.
```

Comment:
I would elaborate a little bit. For example: "To achieve representative power measurement results, it is often desirable to disable UART interfaces unless their contribution to overall power consumption is of interest."